backend/src/modules/boards/repositories/board.repository.ts

```diff
@@ -16,4 +16,19 @@ export class BoardRepository
 	getBoard(boardId: string): Promise<Board> {
 		return this.findOneById(boardId);
 	}
+	updatePhase(boardId, phase): Promise<Board> {
+		return this.findOneByFieldAndUpdate(
+			{
+				_id: boardId
+			},
+			{
+				phase
+			},
+			{ new: true },
+			{
+				path: 'team',
+				select: 'name users -_id'
```

Comment:
```suggestion
				select: 'name -_id'
```
Do you need the users field? if not you can remove it

Reply:
Yes, it can be removed